core/ops_metrics.rs

```diff
@@ -85,12 +85,14 @@ impl OpsTracker {
     metrics.ops_completed_async += 1;
   }
 
+  #[allow(unused)]
   pub fn track_unref(&self, id: OpId) {
```

Comment:
Just delete this, I guess?
@AaronO any opinions?

Reply:
I guess that's fine. Right now "unrefed" ops don't take part in `opSanitizer` but actually it might be preferable to include them.